### PR TITLE
Telemetry for resource scan using base class method 

### PR DESCRIPTION
**Pull request by Aboli-msft**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.Framework/Abstracts/ServicesSecurityStatus.ps1 [hunk 1]
```diff
@@ -175,6 +175,9 @@ class ServicesSecurityStatus: AzSVTCommandBase
 					$this.PublishCustomMessage(" `r`nChecking resource [$currentCount/$totalResources] ");
 				}
 				
+				#resource scan start AI telemetry
+				$this.ResourceScan($_,"Resource Scan Started");
```
Comment (VishalHaibatpure) [inline src/AzSK.Framework/Abstracts/ServicesSecurityStatus.ps1]: Use already existing event to send resource started event

Comment (Aboli-msft) [reply src/AzSK.Framework/Abstracts/ServicesSecurityStatus.ps1]: done changes as susggested

File: src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1 [hunk 2]
```diff
@@ -45,13 +45,60 @@ class AIOrgTelemetry: ListenerBase {
...
+					if ($isResourceScanStartEndTelemetryEnabled) {
+						if ($resourceContext -ne $null) {
+							$resourceDetails = @{
+								ResourceId            = $resourceContext.ResourceId
+								ResourceName          = $resourceContext.ResourceName
+								ResourceType          = $resourceContext.ResourceType
+								Location              = $resourceContext.Location
+								ResourceGroupName     = $resourceContext.ResourceGroupName
+								SubscriptionId        = $Event.SourceArgs[0].SubscriptionContext.SubscriptionId
+								PartialScanIdentifier = $Event.SourceArgs[0].PartialScanIdentifier
```
Comment (VishalHaibatpure) [inline src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1]: $Event.SourceArgs[0].PartialScanIdentifier 
PartialScanIdentifier will not be present in SDL/CICD scan. Please check before accessing it

File: src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1 [hunk 3]
```diff
@@ -45,13 +45,60 @@ class AIOrgTelemetry: ListenerBase {
...
```
Comment (VishalHaibatpure) [inline src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1]: $Event.SourceArgs[0].PartialScanIdentifier 
PartialScanIdentifier will not be present in SDL/CICD scan. Please check before accessing it

File: src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1 (hunk 2 above)
Comment (Aboli-msft) [reply src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1]: Handled it with 'NA' as default value

File: src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1 (hunk 3 above)
Comment (Aboli-msft) [reply src/AzSK.Framework/Listeners/RemoteReports/AIOrgTelemetry.ps1]: Handled it with 'NA' as default value

**Pull request merged**